src/vstart.sh

```diff
@@ -1132,13 +1133,15 @@ fi
 echo "started.  stop.sh to stop.  see out/* (e.g. 'tail -f out/????') for debug output."
 
 echo ""
-if $with_mgr_dashboard; then
-    echo "dashboard urls: $DASH_URLS"
-    echo "  w/ user/pass: admin / admin"
+if [ "$new" -eq 1 ]; then
+    if $with_mgr_dashboard; then
+        echo "dashboard urls: $DASH_URLS"
```

Comment:
... not without refactoring a larger section of ```vstart.sh``` (for example, just running ```vstart.sh``` w/o arguments will drop the number of running daemons). I figure any dashboard developer has obviously not been running ```vstart.sh -k``` since otherwise they would have noticed they broke vstart for other folks.

Reply:
@sebastian-philipp you can always use "ceph mgr services" to get URIs of running modules.  The URI-printing bits in vstart predated that command.